10-angular/ui_router/readme.md

```diff
@@ -0,0 +1,389 @@
+# Angular and $uiRouter
+
+## Learning Objectives
+
+- Explain what dependency injection is and what problem it solves
+- Explain the purpose of templates in Angular
+- Create separate views and routes for each CRUD action
+- Use the ng-view directive to load angular templates
+- Use $stateProvider and $state to access query parameters and update the URL
+- Define multiple controllers in a single module
+
+# Building an actual Angular app
+
+...is going to be what we start doing today. So far, we've pretty much just played around with some of Angular's different built-in components. Now we're going to start writing our own components.
+
+# Getting set up
+
+Please go to this link and download the ZIP file:
+
+https://github.com/ga-dc/grumblr_angular/releases/tag/1.0.0
+
+Then, if your computer didn't automatically "unzip" the ZIP file, double-click it. This should create a folder called `grumblr_angular-1.0.0`. Move it into your `wdi/in-class` folder.
+
+This has the same end result as `git clone` except there's no `.git` folder in there. The reason we're asking you to use this ZIP file is because it gives you a nice little folder with its own version number: the subsequent folders you download will have their own numbers as well, so you won't have a zillion duplicate folders.
```

Comment:
- As far as I know, yes. @jshawl ?
- That it's the starter code of the first lesson. The solution code of the third lesson will be 3.0.1.